src/AppBundle/Resources/config/doctrine/Media.orm.yml

```diff
@@ -35,4 +35,9 @@ AppBundle\Entity\Media:
             joinColumn:
                 name: restaurant_id
                 referencedColumnName: id
+
+    manyToMany:
+        postsMedias:
```

Comment:
maybe you should refer it to "posts" instead of "postsMedias" ? Not sure about this though.